### PR TITLE
proposal for module PR template

### PR DESCRIPTION
**Pull request by wouterdb**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# Merge procedure
+
+Don't use the github built-in merge, but the process described [here](https://docs.internal.inmanta.com/topics/tasks/commiting_changes_modules.html)
+
+```sh
+git pull
+git checkout master
+git pull
+git merge --squash issue/{issue-number}-{short description}
+inmanta module commit -m "{Commit Message Here}" -r
```
Comment (arnaudsjs) [inline .github/pull_request_template.md:16]: The `git add` part is missing here. Or we should do `inmanta module commit -a`.

**Comment by wouterdb** [comment]
@bartv done!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+git push
+git push {tag} # push the tag as well
+```
+
+The close the PR with a reference to the commit
```
Comment (sanderr) [inline .github/pull_request_template.md:21]:
```suggestion
Then close the PR with a reference to the commit
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+# Self Check:
+
+Strike through any lines that are not applicable (`~~line~~`) then check the box
+
+- [ ] Attached issue to pull request
+- [ ] Changelog entry
+- [ ] Proper dev version is set in `project.yml`
+- [ ] Version number is bumped to dev version
```
Comment (sanderr) [inline .github/pull_request_template.md:30]: How does this differ from the one above?

Comment (wouterdb) [reply .github/pull_request_template.md:30]: Mhh, duplicate, thx.

**Pull request merged**